### ISSUE TITLE
Redirect HTTP to HTTPS

### ISSUE THREAD

**Issue by benkebdani** [widdix/mastodon-on-aws#15]
With the current configuration, HTTP traffic is not automatically redirected to HTTPS. Does this have to be handled with listeners on the load balancer?

**Comment by andreaswittig** [comment]
Good point, @benkebdani. I've added an HTTP redirect (see https://github.com/widdix/mastodon-on-aws/commit/b354001a21d3aaa82355c53ea61f2614841a43e0). Please update to `v0.7.0`.
